### DISCUSSION TITLE
how to make a new base for a country, Site Selection, Funding

### DISCUSSION THREAD

**Discussion by shirishag75** [mars-sim/mars-sim#438] [Q&A]
How do I make a new base with some name for a country ? I know we would have to some budgeting that would need to be worked out and all. While perhaps budgeting at the initial stage could be negative, at later stages it probably either needs to be neutral or slight positive. I dunno how we are supposed to do that. At the initial stage, being able to have a site name and having few beds etc. would be nice. Please let me know if there is a way to do that without trying to delve into xml stuff.

**Comment by mokun** [comment]
> How do I make a new base with some name for a country ? 

In the `Simulation Configuration Editor`, click `Add` to add a new base. 

You can define the name of your base and pick the right `settlement template`. 

You can choose what latitude and longitude and # of population and # of bot.

And at last you can choose the sponsoring space agency of this settlement.

But for now, there are only 28 [countries](https://github.com/mars-sim/mars-sim/wiki/Countries) that we account and their names are hardcoded. 

That's because they either have their own space agency or are a part of EU. May be we can change that later.

**Comment by mokun** [comment]
> I know we would have to some budgeting ... While perhaps budgeting at the initial stage could be negative, at later stages it probably either needs to be neutral or slight positive.

When you say *budgeting*, do you mean the funding ?

**Comment by mokun** [comment]
> At the initial stage, being able to have a site name 

Do you mean the site selection of a landing ?

> and having few beds etc. would be nice. 

Do you mean which quarters/bed for housing which settler ?

**Comment by shirishag75** [comment]
Yup, by budgeting I meant funding. 

Yeah, having a site name or site selection. From what I recall especially when site was thought for Opportunity and Pathfinder,  there was lot of mapping beforehand. Something like what is being done by Chandrayaan 2 for the moon to soft-land Chandrayaan 2 on Moon. https://www.thehindu.com/sci-tech/science/chandrayaan-2-and-every-other-moon-mission-mapped/article28772260.ece 

So, it would be interesting and exciting for the player to have selection of sites I believe where it could be risky to land and interesting to do science or play it safe depending upon both the player, his funding, insurance etc. While at the beginning I wouldn't want to put all of that but maybe either at hardcore or medium difficulty such things could be thought of. 

Yeah, I mean quarters/bed for housing for earthborns to settle in mars.

**Comment by mokun** [comment]
> So, it would be interesting and exciting for the player to have selection of sites I believe where it could be risky to land and interesting to do science or play it safe

In terms of *Site Selection*, we currently don't have any game mechanics dealing with that. 

We'll need to build rules from the ground up. 

We'll need to write codes that give players the ability to send satellites and robotic probes to scan whatever area that are optimal for landing and define what merit for a particular site to be chosen.

**Comment by mokun** [comment]
>  his funding, insurance etc.

Funding is something else. In that case, we'll need to do more concept exploration and discuss what rules to establish in order to fund each fiscal year for each space agency.

We'll need to put together mechanics for researching and purchasing hardware and software and send what resources to Mars, based on available funding.

We will add these features to mars-sim one by one. It's exciting that players will have a lot more to consider in a simulation.
